src/pages/api/gateway/[gatewayUID]/name.ts

```diff
@@ -25,7 +25,10 @@ function validateRequest(
   req: NextApiRequest,
   res: NextApiResponse
 ): false | ValidRequest {
-  const { gatewayUID, name } = req.query;
+  const { gatewayUID } = req.query;
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  const { name } = req.body;
```

Comment:
To delete this warning, join the [conversation](https://bluesinc.slack.com/archives/C02LAQLMJR5/p1646952266854589) on data validation.